core/src/test/java/org/opentosca/toscana/core/BaseJUnitTest.java

```diff
@@ -1,10 +1,34 @@
 package org.opentosca.toscana.core;
 
+import java.io.File;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.rules.TemporaryFolder;
 
+/**
+ * If files need to get written on disk during tests, use {@link #temporaryFolder}
+ */
 public abstract class BaseJUnitTest extends BaseTest {
```

Comment:
Forgot this in the last review. We should add  `@RunWith(CategoryAwareJUnitRunner.class)`. Does not make any sense to not make the regular unit tests not category aware!

Reply:
fixed